app/src/main/java/sensors_in_paradise/sonar/screen_recording/camera/CameraManager.kt

```diff
@@ -252,35 +250,59 @@ class CameraManager(
             bindImageAnalyzer()
         }
 
-        if (shouldRecordPose()) {
```

Comment:
why can we remove this if() ?

Reply:
It turned out to be unnecessary, because startRecordingPose only gets called if shoudRecordPose()